uint8 BOX=0
uint8 CYLINDER=1
uint8 TORUS=2
uint8 MESH_RESOURCE=3

uint8 INSERT=0
uint8 ERASE=1
uint8 ERASEALL=2
uint8 ERASEFOCUS=3
uint8 COPY=4

int32 type
int32 action
string frame_id
string name
string description
string mesh_resource
bool mesh_use_embedded_materials